<?xml version="1.0" encoding="utf-8"?>
<launch>
	<!-- <arg name="veh" default="husky1" /> -->
	<arg name="x_range" default="10" />
	<arg name="y_range" default="10" />
	<arg name="z_range" default="20" />
	<arg name="voxel_size" default="1" />
	
	<node pkg="mapping" type="pub_loam_tf.py" name="pub_loam_tf" output="screen">
	</node>

	<group ns="$(arg veh)">

		<node pkg="mapping" type="map_filter" name="map_filter" output="screen">
			<param name="x_range" value="$(arg x_range)" />
			<param name="y_range" value="$(arg y_range)" />
			<param name="z_range" value="$(arg z_range)" />
			<param name="voxel_size" value="$(arg voxel_size)" />
		</node>

	</group>

</launch>
